components/__design__/text.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const Text: StoryObj = {
```

Comment:
Let's rename this export as `Anchor` as we will add other stories in this file.

Reply:
renamed!